src/test/ui/resolve/token-error-correct-3.stderr

```diff
@@ -10,17 +10,17 @@ note: unclosed delimiter
 LL |             callback(path.as_ref(); //~ ERROR expected one of
    |                     ^
 
-error: expected one of `,`, `.`, `?`, or an operator, found `;`
+error: expected one of `)`, `,`, `.`, `?`, or an operator, found `;`
   --> $DIR/token-error-correct-3.rs:24:35
    |
 LL |             callback(path.as_ref(); //~ ERROR expected one of
-   |                                   ^ expected one of `,`, `.`, `?`, or an operator here
+   |                                   ^ expected one of `(`, `,`, `.`, `?`, or an operator here
```

Comment:
I'm confused, why is this the opposite of the error message above?

Reply:
Opposite? I changed the error message because that's the purpose of the issue being solved.